server/migrations/20231120142213-add-column-id-to-group-permissions.js

```diff
@@ -0,0 +1,47 @@
+"use strict";
+
+module.exports = {
+  async up(queryInterface, Sequelize) {
+    await queryInterface.sequelize.transaction(async (transaction) => {
+      await queryInterface.sequelize.query(
+        `CREATE EXTENSION IF NOT EXISTS "uuid-ossp";`,
+        { transaction }
+      );
+      await queryInterface.addColumn(
+        "group_permissions",
+        "id",
+        {
+          type: Sequelize.UUID,
+          defaultValue: Sequelize.literal("uuid_generate_v4()"),
+        },
+        { transaction }
+      );
+      await queryInterface.changeColumn(
+        "group_permissions",
+        "id",
+        {
+          type: Sequelize.UUID,
+          allowNull: false,
+        },
+        { transaction }
+      );
+      await queryInterface.addConstraint("group_permissions", {
+        type: "PRIMARY KEY",
+        fields: ["id"],
+        transaction,
+      });
+    });
+  },
+
+  async down(queryInterface, Sequelize) {
+    await queryInterface.sequelize.transaction(async (transaction) => {
+      await queryInterface.removeColumn("group_permissions", "id", {
+        transaction,
+      });
+      await queryInterface.sequelize.query(
+        `DROP EXTENSION IF EXISTS "uuid-ossp";`,
+        { transaction }
+      );
```

Comment:
```suggestion
```

I don't think we need to fully revert here, if one of these migrations needs to be reverted we don't want to lose that extension